docs/conf.py

```diff
@@ -87,53 +88,12 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = "sphinx_symbiflow_theme"
+html_theme = "sphinx_book_theme"
 
 html_theme_options = {
     # Repository integration
     # Set the repo url for the link to appear
     'github_url': 'https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts',
-    # The name of the repo. If must be set if github_url is set
-    'repo_name': 'OpenROAD Flow',
-    # Must be one of github, gitlab or bitbucket
-    'repo_type': 'github',
-
-    # Set the name to appear in the left sidebar/header. If not provided, uses
-    # html_short_title if defined, or html_title
-    'nav_title': "OpenROAD Flow",
-
-    # A list of dictionaries where each has three keys:
-    #   href: The URL or pagename (str)
-    #   title: The title to appear (str)
-    #   internal: Flag indicating to use pathto (bool)
-    'nav_links': [
-        {"title": "Home", "href": "index", "internal": True},
-        {"title": "The OpenROAD Project", "href": "https://theopenroadproject.org", "internal": False},
-    ],
```

Comment:
Does this new theme have support for something similar? This argument, in the previous theme, added a button to point to the project website.

Reply:
@vvbandeira Made some icons and took out the "How to contact us section"

Reply:
Awesome! Please add to the OR PR as well.